gui/default/syncthing/settings/settingsModalView.html

```diff
@@ -159,6 +159,161 @@
               </div>
             </div>
           </div>
+
+          <!-- WebAuthn settings -->
+          <div class="form-group">
+
+            <!-- WebAuthn credentials -->
+            <div class="table-responsive webauthn-credentials">
+              <table class="table-striped table">
+                <thead>
+                  <tr>
+                    <th colspan="1000">
+                      <label translate>WebAuthn Credentials (Passkeys)</label>
+                      <a href="{{docsURL('users/webauthn')}}" target="_blank">
+                        <span class="fas fa-question-circle"></span>&nbsp;<span translate>Help</span>
+                      </a>
+                    </th>
+                  </tr>
+                  <tr>
+                    <th><label translate>Name</label></th>
```

Comment:
```suggestion
                    <th><label translate="settings.webauthn.credentials.columnHeading.name">Name</label></th>
```

Reply:
Fixed in 4d5b84af835b56f23281a96c3d756ad0d04b846b.